src/main/java/spireMapOverhaul/zones/manasurge/ManaSurgeZone.java

```diff
@@ -0,0 +1,245 @@
+            List<AbstractManaSurgeModifier> commonModifierList = new ArrayList<>(getPositiveCommonModifierList(true));
+            CardModifierManager.addModifier(card, Wiz.getRandomItem(commonModifierList, AbstractDungeon.cardRng));
+
+        } else {
+            List<AbstractManaSurgeModifier> uncommonModifierList = new ArrayList<>(getPositiveUncommonModifierList(true));
+            CardModifierManager.addModifier(card, Wiz.getRandomItem(uncommonModifierList, AbstractDungeon.cardRng));
+        }
+    }
+
+    public static void applyPermanentNegativeModifier(AbstractCard card) {
+        if (AbstractDungeon.cardRng.randomBoolean(COMMON_CHANCE)) {
+            List<AbstractManaSurgeModifier> commonModifierList = new ArrayList<>(getNegativeCommonModifierList(true));
+            CardModifierManager.addModifier(card, Wiz.getRandomItem(commonModifierList, AbstractDungeon.cardRng));
+        } else {
+            List<AbstractManaSurgeModifier> uncommonModifierList = new ArrayList<>(getNegativeUncommonModifierList(true));
+            CardModifierManager.addModifier(card, Wiz.getRandomItem(uncommonModifierList, AbstractDungeon.cardRng));
+        }
+    }
+
+    @Override
+    public AbstractZone copy() {
+        return new ManaSurgeZone();
+    }
+
+    @Override
+    public void atBattleStartPreDraw() {
+        Wiz.applyToSelf(new ManaSurgePower(AbstractDungeon.player,0));
+    }
+
+    @Override
+    public void modifyRewardCards(ArrayList<AbstractCard> cards) {
+        for (AbstractCard card : cards) {
+            if (card.cost != -2 && card.type != AbstractCard.CardType.CURSE && card.type != AbstractCard.CardType.STATUS) {
+                applyRandomPermanentModifier(card);
+            }
+        }
+    }
+
+    @Override
+    public void postAddButtons(ArrayList<AbstractCampfireOption> buttons) {
+        buttons.add(new EnchantOption(!CardGroup.getGroupWithoutBottledCards(AbstractDungeon.player.masterDeck.getPurgeableCards()).isEmpty()));
+    }
+
+    @Override
+    public void postCreateShopCards(ArrayList<AbstractCard> coloredCards, ArrayList<AbstractCard> colorlessCards) {
+        for (AbstractCard card : coloredCards) {
+            if (card.cost != -2 && card.type != AbstractCard.CardType.CURSE && card.type != AbstractCard.CardType.STATUS) {
+                applyRandomPermanentModifier(card);
+            }
+        }
+        for (AbstractCard card : colorlessCards) {
+            if (card.cost != -2 && card.type != AbstractCard.CardType.CURSE && card.type != AbstractCard.CardType.STATUS) {
+                applyRandomPermanentModifier(card);
+            }
+        }
+    }
+
+    @Override
+    public float modifyCardBaseCost(AbstractCard c, float baseCost) {
+        if (hasManaSurgeModifier(c)) {
+            if (isCommonModifier(c)) {
+                if (isPositiveModifier(c)) {
+                    return baseCost + 40;
+                } else {
+                    return baseCost - 20;
```

Comment:
Why is the cost reduction less than the increase? Aren't the blights just as bad as the enchatments?

Reply:
I originally had both the same but even with the Blight it felt weird for some cards to roll at like 5-10g considering they can still be useful. I can definitely change it if you think it'd be better. What would you suggest? just +40 -40?

Reply:
I'd say we go for it. The uncommon modifiers are really bad (losing 1 str/dex on a non power/exhausting card is crippling) having that card be cheap or near free would at least make it a viable pickup, I think.

Yeah, I#d say go for the same cost reduction as the opposite increase if it had been positive.

Reply:
done and pushed